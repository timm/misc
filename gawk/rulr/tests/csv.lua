local l=dofile("../src/rulr.lua").lib


local c,n = 0,0
for t in l.csv("auto93.csv") do
  n = n+1
  c = c + #t
  if n % 30==0 then print(n, l.o(t)) end end

assert(c==3192,"bad read")
