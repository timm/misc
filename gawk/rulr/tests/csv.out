% lua csv.lua

        	{Clndrs Volume HpX Model origin Lbs- Acc+ Mpg+}
        	{8 302 129 75 1 3169 12 10}
        	{8 318 150 72 1 4135 13.5 20}
        	{6 168 120 76 2 3820 16.7 20}
        	{3 70 90 73 3 2124 13.5 20}
        	{6 232 90 78 1 3210 17.2 20}
        	{6 231 110 75 1 3039 15 20}
        	{6 173 110 81 1 2725 12.6 20}
        	{4 140 92 76 1 2572 14.9 30}
        	{4 97 88 72 3 2100 16.5 30}
        	{4 90 70 76 2 1937 14.2 30}
        	{4 85 65 79 3 2020 19.2 30}
        	{4 98 65 81 1 2045 16.2 30}
        	{4 85 70 78 3 2070 18.6 40}
